src/main/java/com/kodilla/ecommercee/domain/Group.java

```diff
@@ -11,11 +11,15 @@
 @AllArgsConstructor
 @NoArgsConstructor
 @Getter
-@Entity
+@Entity(name = "PRODUCT_GROUP")
```

Comment:
Dlaczego PRODUCT_GROUP?

Reply:
Group to słowo kluczowe SQL, więc nie może być nazwą tabeli.